### DISCUSSION TITLE
how to download anime without show_name in folder?

### DISCUSSION THREAD

**Discussion by candrapersada** [sdaqo/anipy-cli#218] [Q&A]

**Comment by sdaqo** [comment]
you can not do that currently and I also do not know why you would do that, either way there is a ongoing discussion: #215 in which a feature is being discussed that could enable you to do this.

**Comment by candrapersada** [comment]
`download/Blue Lock vs U-20 Japan/Blue Lock vs U-20 Japan_01`
to `download/Blue Lock vs U-20 Japan_01`

**Comment by sdaqo** [comment]
Closest shot is #162, but again #215 is also promising as this would remove the need for running the script externally
